docs/howto/update.md

```diff
@@ -52,6 +52,12 @@ func (q *Queries) UpdateAuthorBios(ctx context.Context, bio string) error {
 If your query has more than one parameter, your Go method will accept a
 `Params` struct.
 
+```sql
+-- name: UpdateAuthor :exec
+UPDATE authors SET bio = $2
+WHERE id = $1;
+```
```

Comment:
Although somewhat obvious, it would be helpful to have SQL examples for Multiple parameters as well as Single parameters.